File: isPerfectSquare.js
const isPerfectSquare = input => {

    if (input <= 0) return false;

    const SQUARE_ROOT = parseInt(input ** 0.5);
    return SQUARE_ROOT * SQUARE_ROOT === input;
};

console.log( isPerfectSquare(4) ); //true
console.log( isPerfectSquare(5) ); //false

File: is_perfect_square.py
import sys
def is_perfect_square(input):
    # Square root of input
    sqrt = int(input ** 0.5)
    # If square root is equal to input
    if sqrt * sqrt == input:
        return True
    else:
        return False

sys.modules[__name__] = is_perfect_square
